Hai Reveal: Council Proceedings (A11) - encourage readership by increasing replay value (#8212)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

				set "Hai Reveal [A11] Sound the Alarm: pirates book"
				"Hai Reveal [A11] Sound the Alarm: time" ++
			`	The report begins with a brief summary of Hai involvement with anarchist groups, which dates back from the founding of the first anarchist colony, Freedom, around the same time as human first contact. Apparently, the Hai have had generally positive relations with the largest organizations on anarchist planets. The Hai trade humanitarian supplies or other goods for assistance from the anarchist planets, even ones as distant as Thule and Greenrock.`
			`	Much of the nature of the "assistance from anarchist planets" is redacted, but it seems it mostly involves shutting down groups that try to reveal or harm the Hai. However, there are notes about deals with anarchists who oppose slavery, arranging massive rescues, bringing thousands of slaves at a time to freedom in Hai space. Also, there is an amnesty program for pirates who have committed no serious crimes, which explains the abundance of reformed pirates on Allhome.`

> 			`	Recent anti-pirate actions by the Free Worlds have weakened the large anarchist organizations that had deals with the Hai, and the Republic is expected to follow suit. This hindered the Hai ability to control pirates' activities near Ultima Thule, leading to the current crisis. Previous approaches that worked with large organizations, haven't worked with small pirate groups.`

Reviewer: ```suggestion
			`	Recent anti-pirate actions by the Free Worlds have weakened the large anarchist organizations that had deals with the Hai, and the Republic is expected to follow suit. This hindered the Hai ability to control pirates' activities near Ultima Thule, leading to the current crisis. Previous approaches that worked with large organizations have not worked with small pirate groups so far.`
```

			`	They can't track small pirate groups, but they clearly have an intelligence network. Is it good for anything?`
				goto reading
				to display
					not "Hai Reveal [A11] Sound the Alarm: human book"

> 			`	It seems that, although the Hai have no trouble with large, established, hierarchies, they have trouble tracking small, independent, anarchist, groups. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`

Reviewer: ```suggestion
			`	It seems that, although the Hai have no trouble with large, established hierarchies, they have trouble tracking small, independent, anarchist groups. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`
```

			`	The remaining 80% of the report is redacted in its entirety, save for a few section headings:`
			`	-	Multi-Decadal Timeline Expected before Unfettered Progress`
			`	-	Choot'k: Insane Risk-Taker or Brilliant Strategist?`
			`	-	Reports from Hai Agents on Unfettered Raiding Vessels`
			`	-	Lack Of Success Spying on Unfettered Military Leadership`
			`	-	Theories on the Sentient Evolution of Nocturnal Raptors`
			`	Musing on the titles, you recall that "raptors" are birds of prey. Why would an intelligence report on the Unfettered discuss nocturnal birds of prey?`
				goto reading

			label acorns
			action

> 				log `A Hai intelligence report said Unfettered planets have been ruined to the point where they cannot only produce enough food for bare sustenance. Birds, predators, and large species were almost wiped out, destroying the ecosystem. It mentioned three species of oak that could grow, but none are ideal. As time goes on Unfettered will increasingly rely on external food sources.`

Reviewer 2: ```suggestion
				log `A Hai intelligence report said Unfettered planets have been ruined to the point where they can only produce enough food for bare sustenance. Birds, predators, and large species were almost wiped out, destroying the ecosystem. It mentioned three species of oak that could grow, but none are ideal. As time goes on Unfettered will increasingly rely on external food sources.`
```
double neg was weird

				set "Hai Reveal [A11] Sound the Alarm: acorns book"
				"Hai Reveal [A11] Sound the Alarm: time" ++
			`	This report describes the rapid degradation of the environmental condition on Unfettered worlds, and frequent problems with crop blight. This comes from over-farming, lack of pollution controls, over-mining, and simply a lack of resources to handle unexpected environmental catastrophes. Though the Unfettered always have enough food produced internally for bare sustenance, a civilization needs more than that. They stockpile food during excess, including by raids or aid. Such stockpiles run out every year or two during severe catastrophes, causing more raids and necessitating more aid.`

> 			`	There is a detailed study on alternative crops, noting that the most efficient food for Hai is nuts, especially acorns, just like squirrels on Earth. It notes that common evolutionary pressures lead to common solutions, listing oak tree species on various planets. Apparently Earth's willow oak would grow well on Warfeed, but the powerful aphrodisiac effects, "may have unintended consequences." Also, the "Mile-High Blue Oak" is a rapid food source, but it takes a century to begin producing acorns. The Hai are still investigating possible crops, beyond the ones the Unfettered have already accepted.`

Reviewer: ```suggestion
			`	There is a detailed study on alternative crops, noting that the most efficient food for Hai is nuts, especially acorns, just like squirrels on Earth. It notes that common evolutionary pressures lead to common solutions, listing oak tree species on various planets. Apparently Earth's willow oak would grow well on Warfeed, but the powerful aphrodisiac effects "may have unintended consequences." Also, the "Mile-High Blue Oak" is a rapid food source, but it takes a century to begin producing acorns. The Hai are still investigating possible crops, beyond the ones the Unfettered have already accepted.`
```

				set "Hai Reveal [A11] Sound the Alarm: teeneep1 book"
				"Hai Reveal [A11] Sound the Alarm: time" ++
			`	This is an extremely biased report of Teeneep's life up to full physical maturity. It focuses on chaotic incidents of her past, and uses them to justify the claim that she has always been a menace to society.`
			`	The most telling event happens shortly after Teeneep's mother mysteriously vanishes on a science expedition to Darkwaste, leaving the young Hai quite stressed. Soon after, at school, she saw a teacher berating a classmate about "unfettered tendencies." In response, she stood on a table and started chanting anti-bullying protest slogans. The whole class joined in, and they marched through the hallways chanting. An hour later, all students in the school was doing it, and they locked themselves in a well-stocked cafeteria, refusing to come out until the teacher was fired.`
			`	Two days later, the offending teacher was unemployed, and Teeneep was in a "mandatory juvenile behavior modification program" which the report says she was "failing spectacularly."`

> 			`	Also, throughout the report, there were complaints about Teeneep befriending "problematic members of society." This included former pirates, Unfettered Hai, freed slaves, and even reformed violent convicts. The report admits Teeneep generally had a positive impact on the lives she touched, but insists that her tendency to enter "that part of society" is a major concern. In particular, it notes that her quite athletic combat abilities developed in her youth during her time with "troubled individuals."`

Reviewer: ```suggestion
			`	Also, throughout the report there were complaints about Teeneep befriending "problematic members of society." This included former pirates, Unfettered Hai, freed slaves, and even reformed violent convicts. The report admits Teeneep generally had a positive impact on the lives she touched, but insists that her tendency to enter "that part of society" is a major concern. In particular, it notes that her quite athletic combat abilities developed in her youth during her time with "troubled individuals."`
```

			`	It warns the growing movement has become a threat to the stability of both Hai and human society.`
				goto reading

			label quarg
			action
				log `A Hai intelligence report reports a mutual fondness between Hai and Quarg for one anothers' cuisine. It also mentioned lingering frustration about the Quarg refusing to fight the Korath during their warlike spiral that led to near extinction. The Hai felt that Quarg involvement would have saved more Korath lives.`
				set "Hai Reveal [A11] Sound the Alarm: quarg book"
				"Hai Reveal [A11] Sound the Alarm: time" ++

> 			`	This report is surprisingly light on details of any interest. It seems the Hai view the Quarg as a nearby race of friendly neighbors. There's frequent cultural exchanges, and a large food trade in both directions. Apparently, "genuine Hai cuisine" is popular among the Quarg, especially the famous Hai acorn cookies. In return, the Hai use Quarg spices extensively, including some of the meals you've eaten since you arrived. The spices are described as, "subtle and mysterious."`

Reviewer: ```suggestion
			`	This report is surprisingly light on details of any interest. It seems the Hai view the Quarg as a nearby race of friendly neighbors. There's frequent cultural exchanges, and a large food trade in both directions. Apparently, "genuine Hai cuisine" is popular among the Quarg, especially the famous Hai acorn cookies. In return, the Hai use Quarg spices extensively, including some of the meals you've eaten since you arrived. The spices are described as "subtle and mysterious."`
```

				`	QRG-3Z-18FJ4KC: Modern Hai-Quarg Relations`
					goto quarg
					to display
						not "Hai Reveal [A11] Sound the Alarm: quarg book"
				`	HUM-A0C1I3X: Comparisons of Hai and Human Military Technology`
					goto militech
					to display
						not "Hai Reveal [A11] Sound the Alarm: militech book"
				`	"I've changed my mind. Take me back to the proceedings instead."`
					goto reading
					to display
						"Hai Reveal [A11] Sound the Alarm: time" == 2
				`	"I'm taking a nap. Let me know when it's all over."`
					goto nap
					to display
						"Hai Reveal [A11] Sound the Alarm: time" > 2

			label solutions
			action
				set "Hai Reveal [A11] Sound the Alarm: solutions book"
				"Hai Reveal [A11] Sound the Alarm: time" ++
				log `A Hai intelligence report identifies three problems: "northern troubles," degrading secrecy of the Hai, and piracy. They expect piracy to vanish from the Republic "befriending" anarchist colonies and suggest Hai slowly reveal themselves through leaked information. It is unclear if that ever happened. For the "northern troubles," they have a "skilled mercenary" in mind for the job, and a diplomat.`

> 			`	This one-page report is recent, and includes references to other reports you can access.`

Reviewer 2: I think that we should get a mention of it being stored in the logs?
Or is the idea FL just remembers about this? Or logs this like all other logs?

Author: However the other logs are stored is how these are stored.

Reviewer 2: Fair. I was just pointong out you could somehow tell it to the player
"You took good notes of the reportw in your logs" when ur done reading for instance

			`	Of course, if their intelligence agencies are so capable, shouldn't they have been able to solve their pirate problems?`
				goto reading
				to display
					not "Hai Reveal [A11] Sound the Alarm: pirates book"
			`	It seems that, although the Hai have trouble tracking small, independent, anarchist, groups, they have no trouble with large, established, hierarchies. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`
				goto reading
				to display
					has "Hai Reveal [A11] Sound the Alarm: pirates book"

			label pirates
			action
				log `The Hai intelligence agencies' approach to pirates was to befriend the large anarchist groups, as if they were the head of an "anarchist hierarchy. This suggests they lack understanding of raw chaos; in essence, their greatest weakness is asymmetric warfare.`
				set "Hai Reveal [A11] Sound the Alarm: pirates book"
				"Hai Reveal [A11] Sound the Alarm: time" ++

> 			`	The report begins with a brief summary of Hai involvement with anarchist groups, which dates back from the founding of the first anarchist colony, Freedom, around the same time as human first contact. Apparently, the Hai have had generally positive relations with the largest organizations on anarchist planets. The Hai trade humanitarian supplies or other goods for assistance from the anarchist planets, even ones as distant as Thule and Greenrock.`

Reviewer 2: Wait what?
This is new and totally unexpected to me.. How do they keep their cover? In human ships?
The Unfettered trade with anarchists but it generally goes the other way, and I wouldnt expect the Hai to approve although this would facilitate the job of those who want to smuggle to unfet space

Author: This is discussed elsewhere in Hai Reveal. Whatever method they use to spy on the human authorities, is likely how they contact the pirates. The simplest option is just sending encrypted messages through the hyperspace relays. However, they do have contact with human merchants who can be paid to smuggle messages. Also, we know they can hide satellites in a manner that prevents humans from finding them (the hyperspace relay mission), probably by making them look like asteroids or similar debris.

So, yes, the txt already refers to them doing this, and there are many ways they could accomplish it.

Author: Also, the Unfettered have a lot of experience hiding their doings from the Fettered. I would expect special ops missions and smuggling operations to human space would be no exception to this.

Reviewer 2: Well the unfet are in their own territpry, this is said to be done by Hai outside of theirs

Author: Yes, and the Hai on both sides of Wah Ki have a hundred years of experience hiding their doings from humans.

Reviewer 2: Yes, by acting with proxies?

Author: I'm not revealing how the Hai intelligence service works in these reports. They are low-level (hazel-2) reports, which only provide results of the intelligence, not sources or capabilities.

Author: This is now explained in the PR description, "You only have hazel-2 clearance, which does not let you learn how the Hai intelligence service works; you get the results, but not the sources or capabilities."

> 			`	Much of the nature of the "assistance from anarchist planets" is redacted, but it seems it mostly involves shutting down groups that try to reveal or harm the Hai. However, there are notes about deals with anarchists who oppose slavery, arranging massive rescues, bringing thousands of slaves at a time to freedom in Hai space. Also, there is an amnesty program for pirates who have committed no serious crimes, which explains the abundance of reformed pirates on Allhome.`

Reviewer 2: I'd love if this was presented another way: they found the old pirates to help their own, given they know the place and stuff and wont attract any attention

Author: Hai Reveal already mentions a pattern of funding large anarchist organizations. I'm merely elaborating on that point.

Reviewer 2: Yeah and I think it'd be smarter to do it by founding old pirates

Author: The intelligence reports do not describe how the Hai achieve what they do. This is low-level intelligence, which only provides results, not sources.

> 			`	Recent anti-pirate actions by the Free Worlds and Republic Navy have weakened the large anarchist organizations that had deals with the Hai. This hindered the Hai ability to control pirates' activities near Ultima Thule, leading to the current crisis. Previous approaches that worked with large organizations, haven't worked with small pirate groups.`

Reviewer 2: Did it? The war happened so FW was saying anarchist had more power than ever, unlike the republic which is having a hard time defending itself

Author: No, this is post-war. The Free Worlds removed pirates from several systems by befriending the local anarchist groups and having them kick the pirates out. This has reduced piracy in the Free Worlds to the lowest level in quite a few years. Hai Reveal A14 informs you that the Republic is about to do the same. That's what the paragraph refers to.

Reviewer 2: Then mb focus on that part; say it was those FW pirates?

Author: Oh! I understand my mistake now. Yes, the Republic hasn't done that yet; they let pirates run rampant. I have corrected that. How does it look now?

				goto reading
				to display
					not "Hai Reveal [A11] Sound the Alarm: human book"
			`	It seems that, although the Hai have no trouble with large, established, hierarchies, they have trouble tracking small, independent, anarchist, groups. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`
				goto reading
				to display
					has "Hai Reveal [A11] Sound the Alarm: human book"

			label unfettered
			action
				log `A Hai intelligence report puts the cost to Hai of the Unfettered raids at 3.5 billion credits a day, while the cost is expected to double in "full Hai unification." Unification costs include 5 billion credits a day of "military aid."`
				log `A Hai intelligence report on "Unfettered military activities" included several fully-redacted sections, including "Multi-Decadal Timeline Expected before Unfettered Progress," and "Theories on the Sentient Evolution of Nocturnal Raptors."`
				set "Hai Reveal [A11] Sound the Alarm: unfettered book"
				"Hai Reveal [A11] Sound the Alarm: time" ++

> 			`	This report begins with an economic analysis of the cost of raids at Wah Ki, which it places at a staggering 3.5 billion credits per day. This is 1.5 billion impact on trade; 1.5 billion for military fleet repairs, replacements, ammunition, and infrastructure to support it; and an estimated 500 million in humanitarian aid to the Unfettered, to prevent the problem from worsening. It compares that to "full Hai unification" for which the cost is 7 billion credits per day: 3 billion in humanitarian aid to the Unfettered, 5 billion in military aid, but 1 billion per day recouped through improved trade.`

Reviewer 2: How does unification cost more? What is this unification?
Hai+unfettered if unfettered invade?

Author: Unification means no barriers to funding the Unfettered. The Unfettered military is the Hai military again (or, at least, the most competent part of it) and they'll properly funded as such. The Fettered have the responsibility for paying all social welfare costs of the Unfettered. That's expensive. However, removing the barrier to trade at Wah Ki will make trade more efficient. This is roughly patter ended off of Korean Unification, which has similarly staggering costs (albeit not military).

Reviewer 2: Idea is ok it prob could be clarified though

Author: It could be clarified, but it shouldn't be. That report is supposed to be vague and ominous.

			`	The remaining 80% of the report is redacted in its entirety, save for a few section headings:`
			`	-	Multi-Decadal Timeline Expected before Unfettered Progress`
			`	-	Choot'k: Insane Risk-Taker or Brilliant Strategist?`
			`	-	Reports from Hai Agents on Unfettered Raiding Vessels`

> 			`	-	Unsustainable Rate of Losses Spying on Unfettered Leadership`

Reviewer 2: ```suggestion
			`	-	Lack Of Success Spying on Unfettered Military Leadership`
```
I feel like it wouldnt be that many losses, more like the fact you cant get any idea of whats going on deep

Lemme explain: the first unfet mission shows Chiefs (Uhai Elders) gathered and answering the populace on many issues
That is smth anyone can get on the unfettered.

However, military stuff would be closely garded. I dont think the agents would be dumb enough to enter the throne rooms of warlords to get intel (at that point they would die yes) but instead would be smarter than that and satisfied with (a bit) of second hand intel?

If you really want to press the matter then it makes sense they would take risks and die but not otherwise
Mb adding recent could work, at the very least Military Leadership is needed

Author: The success is limited to sneaking agents onto raiding vessels.

Author: I applied your change, but I'm leaving this open for further discussion.

Reviewer 2: You dont get on a raiding vessel without deserving it though
But I can accept the agents being undercover for q long time

this would create frictions though, since the unfetteted would likely sometimes learn about it, which is the opposite of what they want?

Author: The Unfettered expect the Fettered to spy on them, and vice-versa. It is how cold wars work. I'm sure they scold each other whenever it happens and go back to the status quo.

Reviewer 2: Alr well its ok since the text was changed - they shouldnt outright tell you they're failing because their agents are dying anyway

Author: Yes, you are correct, they should not say that. However, I do want them to warn you that they don't really know what the Unfettered are planning. That's why they don't redact the "Lack Of Success Spying on Unfettered Military Leadership" section heading.

			`	-	Theories on the Sentient Evolution of Nocturnal Raptors`
			`	Musing on the titles, you recall that "raptors" are birds of prey. Why would an intelligence report on the Unfettered discuss nocturnal birds of prey?`
				goto reading

			label acorns
			action
				log `A Hai intelligence report said Unfettered planets have been ruined to the point where they cannot only produce enough food for bare sustenance. Birds, predators, and large species were almost wiped out, destroying the ecosystem. It mentioned three species of oak that could grow, but none are ideal. As time goes on Unfettered will increasingly rely on external food sources.`
				set "Hai Reveal [A11] Sound the Alarm: acorns book"
				"Hai Reveal [A11] Sound the Alarm: time" ++

> 			`	This report describes the catastrophic environmental collapse of the Unfettered worlds through over-farming, lack of pollution controls, and over-mining. Nearly all large species, including predators and nearly all birds, have been wiped out, ruining the ecosystems. Presently, the Unfettered can produce slightly more food needed for bare sustenance on their own, and are reliant on donations or raids for the rest.`

Reviewer 2: The Unfettered situation is bad but not *that bad*
Most food prod happens on one planet (Warfeed), and in normal times it is barely ok.
The issue is there often are events causing the situation to degenerate (a blight, a natural disaster...) So they dont have any stockpiles and most dont eat as much as they would like in normal times but its ok
I portrayed the food crisies as happening every 500 days and lasting about 150

They raid to have a stockpile to survive during those times

Author: I have updated it. How does it look now?

> 			`	There is a detailed study on alternative crops, mentioning that many worlds have evolved a plant like an oak tree, and some will grow readily on the ruined Unfettered world of Warfeed. The swamp white oak from Earth is known to have healing properties, but is insufficient as a food source. The Earth's willow oak has powerful aphrodisiac and stimulant effects. There is a "Mile-High Blue Oak" species on Featherweight with powerful healing effects and is a rapid food supply, but takes two hundred years to begin producing acorns. This means the Unfettered need another source of food.`

Reviewer 2: I dont mind this but if they knew about this food crisis why did they not attempt a long term solution?
I think saying that could be more interesting than acorns but we could have both
Say such an action was attempted but failed because of smth (likely the unfettered not managing to maintain it in the long term)

Author: This is not supposed to be a comprehensive list of possibilities, but if you have anything specific to suggest, let me know and I'll write it up.

Author: I have updated it based on your suggestions. How does it look now?

Reviewer 2: the phrasing is a bit odd and you havent spoken much of it but its not really needed I guess
and idk about "beyond the ones the unfet have already accepted"

			`	This report is surprisingly light on details of any interest. It seems the Hai view the Quarg as a nearby race of friendly neighbors. There's frequent cultural exchanges, and a large food trade in both directions. Apparently, "genuine Hai cuisine" is popular among the Quarg, especially the famous Hai acorn cookies. In return, the Hai use Quarg spices extensively, including some of the meals you've eaten since you arrived. The spices are described as "subtle and mysterious."`
			`	Between the mundane and cultural, you find a few interesting facts. There is some lingering frustration about the Quarg refusing to help fend off Korath raids. This is partly due to the Hai having to fight for themselves, but mostly out of a belief that more Korath would have survived the war if the Quarg had intervened. Also, the Hai apparently have no interest in developing the jump drives Quarg have in abundance due to "the fate of the Korath," and warnings from the Quarg about the dangers of the jump drive manufacturing process.`
				goto reading

			label drak
			action
				set "Hai Reveal [A11] Sound the Alarm: drak book"
				"Hai Reveal [A11] Sound the Alarm: time" ++

> 			`	Although this report is almost a hundred pages long, there's only about three pages of text. It seems the Hai have no certain knowledge about the Drak, save that they're friends of the Quarg. There are legends in the Mythos of the Dara-Irik, faceless creatures of destruction, but the report notes the similarity in name is likely a coincidence. It also describes "Drak Archons," giant space creatures, whom the Hai credit both for saving the Korath and ending the raids on the Hai. However, they're uncertain about whether the Archons are Drak, are ships that contain Drak, or are merely sentient creations of the Drak.`

Reviewer: > It also describes "Drak Archons," giant space creatures, whom the Hai credit both for saving the Korath and ending the raids on the Hai.

Something about this is off but I can't think of a way to replace it.

Author: I rewrote the report and added the missing log entry. How does it look now?

Reviewer: I should have highlighted that it was really `whom the Hai credit both for saving the Korath and ending the raids on the Hai` that sounded weird. Whose raids on the Hai?

Author: Ah, yes, that is a critical detail, isn't it? Now it clarifies that the Archons ended the Korath raids on the Hai.

Reviewer 3: > Ah, yes, that is a critical detail, isn't it? Now it clarifies that the Archons ended the Korath raids on the Hai.

That is new lore that has never thus far been alluded to, and would also substantially undermine the merits of the Unfettered's position.

Reviewer 2: ~~thx for telling me~~
It probably is what happened, to be fair, and explains why the war ended all of a sudden with the proper ships to respond to the raids not mass produced yet.
But that does not mean some Unfettered won't believe it's thanks to them the raids stopped - or if it was someone else

I think we can go with the idea it was a mystery - I don't see the Archons showing up like "hey we stopped the raids, bye now have a nice day" so they probably learned it from the Quarg, eventually, but not all choose to believe it?

Author: No, no, no. The information in the reports is confidential, and comes from Hai Intelligence. They are aware that the Archons stopped the Korath mass suicide, and that the Korath stopped raiding not long after. It isn't difficult for an intelligence agency to establish a causal relationship.

That does *not* mean that the Unfettered have come to the same conclusion, nor even that the Hai public is aware of the Archons.

Author: I have updated the intelligence report to clarify it is providing the knowledge of Hai Intelligence, not the Unfettered or the Hai public.

Reviewer 2: > Also, the Hai had access to plenty of jump drives from the Korath ships raiding them.

do they? Maybe a few but if they did they would not risk them (unlike the Unfettered), and this was never established either

Author: I can remove the Drak intelligence report if absolutely necessary, but I'd rather leave it in, to fill in the plot holes. I am *not* adding anything to the lore in that report; it's simply inescapable conclusions from available information: the Hai are not morons; if they asked the Quarg what happened to the Korath, they would have gotten an answer; and their military (the Unfettered) *would* have pillaged and use jump drives to investigated. From all of this, it is impossible for the Hai not to know what happened to the Korath. There will still be enough intelligence reports for the mission if I remove this one, though the buffer is getting quite thin.

Author: I removed the Drak intelligence report since I don't feel like arguing about it anymore. There's still one extra report, so the buffer is wearing quite thin.

> 			`	What is most concerning to the Hai is the inferiority of their weapons. They note that the ion cannon is effective against humans only because human ships rarely pack large batteries. Other Hai weapons are only marginally better than human alternatives. Meanwhile, a few plasma cannons and flame-throwers can easily shut down a Shield Beetle. Striking a beetle from 45 degrees to the front with four particle cannons will completely paralyze it until it is disabled. Recent human missiles are quite deadly, as are electron beams and plasma turrets. In all, the Hai are quite fearful of human weapons and question whether they could win an all-out war without Unfettered advancements.`

Reviewer 2: ~~You really want to give the player a manual on how to disable SB?~~
The particle cannon one is a bit too weird and unlikely to happen to be stated here dont you think?

Reviewer 2: I do think the Hai have some more powerful weapons in store though, that would probably not be required to fight unfettered, but in the event of a major conflict they should perhaps mention needing to use them or smth, idk
I imagine stuff like duo-pulse, paralyzer (short range ion shock weapon inspired by them exiles) and longer range pulse too, which they would not mention to you like that but they should probably mention needing to produce them if needed. I imagine the reason why they dont is because those are expensive in terms of materials and the like (which the Unfettered dont care about as much)

Also they should mention their trackers give them an edge given humans have no real counter against those

Author: This is an intelligence report from the Hai to a leader of the Free Worlds (or Syndicate or Republic) and it is intended to give the humans a false sense of security. The fact that it isn't 100% complete or 100% accurate is entirely intentional.

Author: I can add a second report with a rebuttal if you give me an outline. Having the Hai represent themselves as massively overpowered is another reasonable approach to an intelligence report in this situation. Certainly, the weapons they have now are laughable at best.

Reviewer 2: If they want you to underestimate them then they want war? The unfettered's idea are so deep in Hai government then?

Author: I really can't justify the Fettered claiming to have good military tech when their current tech is junk. Perhaps I can contrive another report to put in its place?

Author: I have removed this report, and I will add another in its place when I think of one.

					to display
						or
							has "Hai Reveal: what will happen now?"
							has "Hai Reveal: will the elders listen?"
						not "Hai Reveal: will there be war?"
				`	"What are the other options?"`
					to display
						has "Hai Reveal: will there be war?"
						not "Hai Reveal: what are the other options?"
				`	"Who is Aimet?"`
					to display
						has "Hai Reveal: will there be war?"
						not "Hai Reveal: who is Aimet?"
				`	"What is the message?"`
					goto message
			`	As the beeping intensifies, Teeneep thinks for a moment before responding. "Well, the answer-" but before she can finish, several Hai enter the room, surrounding Sayari. Two armed guards look sternly at the startled Teeneep, while a Hai wearing unusual marks of office bears an intensely unemotional face.`
				goto nervous
			label message
			`	The beeping intensifies and Teeneep shows you the message. "Teeneep is hereby subpoenaed by the full Elder Council for an immediate emergency meeting." No sooner do you read this than Sayari returns, aside several other Hai. Two armed guards look sternly at Teeneep, while a Hai wearing unusual marks of office bears an intensely unemotional face.`
			label nervous

> 			`	A nervous young Hai rushes through the doorway, weaving between people into the middle of the crowd, bringing a welcome end to the standoff. She gives an elegant curtsy, curving her tail's tip in place of a skirt, and pulls a paper out of nowhere. Reading its elegant calligraphy, she says, "By order of the Elder Council, Teeneep will immediately accompany the moderator," she motions to the unemotional Hai, "to the Council Chambers for an emergency meeting. Sayari's presence is also required, but Captain <first> <last> is given an alternative. A functionary," she points to herself, "will escort the Captain, and relay questions in their absence."`

Author: This is needlessly long and confusing: "may decline physical attendance and elect to abide in a secure antechamber."

> 			`	A nervous young Hai rushes through the doorway, weaving between people into the middle of the crowd, bringing a welcome end to the standoff. She gives an elegant curtsy, curving her tail's tip in place of a skirt, and pulls a paper out of nowhere. Reading its elegant calligraphy, she says, "By order of the Elder Council, Teeneep will immediately accompany the moderator," she motions to the unemotional Hai, "to the Council Chambers for an emergency meeting. Sayari's presence is also required, but Captain <first> <last> is given an alternative. A functionary," she points to herself, "will escort the Captain, and relay questions in their absence."`

Reviewer 3: I have simplified the language.

			label correct
			`	As his expression changes from unemotional to impassive, the moderator continues. "Your presence," he says pointedly, "is distasteful to many Elders, and," he pauses, "presents opportunities to others. Those factions offer you a once in a lifetime opportunity: access to confidential files from Hai intelligence services, in a private reading room. You may travel between the proceedings and reading room at your leisure. At the end of the proceedings, our internal classified news network will brief you on the proceedings' results.`

			# SCENE CHANGE

			# People filter out, and Captain Last can choose where to go next (and how quickly)

			action
				"Hai Reveal [A11] Sound the Alarm: time" = 1
			# Randomly choose whether the proceedings or reading room are offered first
			# to avoid consistently biasing players to either one.
			branch haidepart
				random < 50
			action
				set "Hai Reveal [A11] Sound the Alarm: proceedings first"
			label haidepart
			`	Teeneep's expression darkens, but she quickly ensures she has what she needs before the Hai begin to depart. Soon, only you and the functionary remain. "Where to, Captain?"`

> 			label wherego

Author: This is far too long. You need to shorten it a lot.

> 			label wherego

Reviewer 3: Most of the breaks you've put in have 5 lines of text between them. This is only 7, so it's not like it's *excessive*, and it does have a bunch of things to convey.

I had a look around and it's not any longer than some of the longer chunks in some Remnant missions or various parts of the Skadenga saga.

The only part that could be reasonably taken out (or paraphrased) is the "As a witness" bit explaining conduct within the chamber, but the last couple times I had a summary line in there that was like "X explains Y to you" or "You receive a message about Z" - I had Amazinite and folks being like "Please just write out the thing instead."

At the end of the day this is the moderator nudging you to ask the questions of your functionary that will lead to the reading room, but the moderator himself must have plausible deniability sufficient to be absolutely above reproach - and that just takes a few words...

I will say, if this went into a "nothing" choice, where all you got was an alternative dialogue line or something, then I'd be much more inclined to try and do something about it, but it does go directly to *THE* main choice branch of the mission - so I think it's ok to be a bit more measured in getting to that.

> 			label wherego

Author: Very well. I will merge it and then attempt to shorten the sentences without removing the plot.

			# SCENE CHANGE

			# Moderator, guards, Sayari, and an agent return.
			
			label surprise
			# The stone acorn is the emblem of the Hai central government: sturdy, stable, and unmoving. Inedible, as a sign that the Hai would never take necessities from each other. Just like the government, a stone must be polished and cared for to remain pristine.
			`	Teeneep's console beeps softly and an icon of a grey acorn is displayed on the screen. She presses the icon and a message comes up. It's hard to make out the words, but the bold font is obviously in your language.`
			# Fake copy of the prior questions so the player feels abruptly interrupted.
			choice

> 				`	"What will happen now then?"`

Reviewer 4: It's hard to tell where the stop is.
```suggestion
				`	"So what will happen now?"`
```

			`	Teeneep's expression darkens, but she quickly ensures she has what she needs before you are all led to the chambers. This is not the first time you've seen the room, but it is the first time you've seen it full. As you all enter and the security staff take their place, the moderator turns to address you.`
			`	"Now, for the record," he says in a clear and audible tone, "I need you to confirm you are able to act as a witness, and confirm that the threat carried from the Unfettered was indeed real."`
			choice
				`	"That is correct."`
					goto correct
				`	"I can also testify to the human position on the matter."`
			`	The moderator says in a dispassionate tone, "Right, well, I doubt that will be necessary.`
			label correct
			`	"In these proceedings, you are only a witness, here to give testimony. You will wait to speak until you are called to the witness stand. Your presence here is a privilege that can be revoked; you don't need to be in this chamber to testify."`
			choice
				`	"Got it. Sit until called. Speak when spoken to."`
					goto "short explanation"
				`	"Hours of dry proceedings aren't why I got into piloting. Can I wait this out elsewhere?"`

> 					goto "please stop flooding my screen with dry text"

Reviewer 4: This label could be a bit shorter?

Author: The length of the label is self-referential humor, concealed within the game data files.

Reviewer 4: Resolved.

				`	"I admire parliamentarian minutia and political intrigue. Now an expert stands before me! Could you explain in detail?"`
			# The moderator's repetitive speech pattern of beginning new declarations with "Now" is intentional.
			`	The moderator's expression changes from unemotional to impassive. "Very well, approach my station with me and I shall brief you on conduct for this session."`

> 			`	You look around as you traverse the route. Although this is a formal proceeding, being observed and recorded, Elders are still filtering in and muttering to each other. Clearly, this is not yet the main event.`

Reviewer 4: ```suggestion
			`	You look around as you traverse the route. Although this is a formal proceeding, being observed and recorded, the Elders are still filtering in and muttering to each other. Clearly, this is not yet the main event.`
```

			`	There is a small desk in the chamber's central circle, opposite the entry point. It was not occupied during your report on Scar's Legion. Now, the moderator leads you to it and takes a seat, gesturing you to sit on a ledge next to it. The discomfort of your position seems to discourage lengthy arguments with the moderator.`
			`	"Now, as a witness, you will have ample opportunity to speak when it is asked of you. You may raise your hand only to protest a matter of factual inaccuracy. You may not protest on a matter of opinion, or it will undermine your quality as a witness. Process dictates that any matter should be confirmed anyway so there should be no need.`
			`	"Your presence," he continues quietly, "is distasteful to many Elders, and," he pauses, "presents opportunities to others."`

> 			`	His dispassionate expression slips slightly and his tone reveals a hint of compassion. "Even the most hardened Hai politicians would not envy the situation you are in. You presence has great potential to both aid or harm Teeneep's case. Some may think less of you for choosing not to attend. My most solemn duty is to ensure that due representation is given to the facts. If you abide in the antechamber, your words will be relayed truthfully and sensitively. Of this you have my word.`

Reviewer 4: ```suggestion
			`	His dispassionate expression slips slightly and his tone reveals a hint of compassion. "Even the most hardened Hai politicians would not envy the situation you are in. You presence has great potential to both aid or harm Teeneep's case. Some may think less of you for choosing not to attend. My most solemn duty is to ensure that due representation is given to the facts. If you abide in the antechamber, your words will be relayed truthfully and sensitively. Of this you have my word."`
```

Author: Multi-line dialog is not supposed to end with a `"`, except the final paragraph. That is a standard punctuation rule, and also one of the rules of Endless Sky dialog.

Reviewer 4: Resolved.

> 			`	"You may travel between the proceedings and the antechamber at your leisure. In the antechamber is a reading room where you may find... other opportunities available. At the end of the proceedings, you may view our internal classified news network's report on the proceedings' results.`

Reviewer 4: ```suggestion
			`	"You may travel between the proceedings and the antechamber at your leisure. In the antechamber is a reading room where you may find... other opportunities available. At the end of the proceedings, you may view our internal classified news network's report on the results of the proceeding."`
```

Reviewer 4: `The results of the proceeding` is still slightly better.

Author: Please update your suggested change, then. I'll press the "commit suggestion" button on the new suggestion.

Reviewer 4: Done.

				goto "functionary will attend you"

			label "short explanation"

> 			`	"Do not misunderstand your position. Even the most hardened Hai politicians would not envy the situation you are in. You have great potential to both aid or harm Teeneep's case.`

Reviewer 4: ```suggestion
			`	"Do not misunderstand your position. Even the most hardened Hai politicians would not envy the situation you are in. You have great potential to both aid or harm Teeneep's case."`
```

> 			`	"Do not misunderstand your position. Even the most hardened Hai politicians would not envy the situation you are in. You have great potential to both aid or harm Teeneep's case.`

Reviewer 4: ```suggestion
			`	"Do not misunderstand your position. Even the most hardened Hai politicians would not envy the situation you are in. You have great potential to both aid or harm Teeneep's case."`
```

> 			`	He leans in, and whispers in a compassionate monotone. "Your presence is distasteful to many Elders, and," he pauses, "presents opportunities to others. In an antechamber's reading room, they have prepared materials you might find... interesting. You may travel between the antechamber and the proceedings as you please. Should we have any questions for you, a functionary will act as a messenger. At the end of the proceedings, you may view our internal classified news network's report."`

Reviewer 4: ```suggestion
			`	He leans in, and whispers in a compassionate monotone. "Your presence is distasteful to many Elders, and," he pauses, "presents opportunities to others. In the antechamber's reading room, they have prepared materials you might find... interesting. You may travel between the antechamber and the proceedings as you please. Should we have any questions for you, a functionary will act as a messenger. At the end of the proceedings, you may view our internal classified news network's report.`
```

				goto "functionary will attend you"

			label "please stop flooding my screen with dry text"

> 			`	He leans in and whispers in a droll monotone. "Your presence is distasteful to many Elders, and," he pauses, "presents opportunities to others. In an antechamber's reading room, they have prepared materials you might find... interesting. You may travel between the antechamber and the proceedings as you please. Should we have any questions for you, a functionary will act as a messenger. At the end of the proceedings, you may view our internal classified news network's report."`

Reviewer 4: ```suggestion
			`	He leans in and whispers in a droll monotone. "Your presence is distasteful to many Elders, and," he pauses, "presents opportunities to others. In the antechamber's reading room, they have prepared materials you might find... interesting. You may travel between the antechamber and the proceedings as you please. Should we have any questions for you, a functionary will act as a messenger. At the end of the proceedings, you may view our internal classified news network's report.`
```

			label "functionary will attend you"
			`	"Now, I have preparations to make. The functionary will attend to you for anything further." He dismisses you, but the functionary intercepts you before you can rejoin Teeneep and Sayari.`

			# SCENE CHANGE
			# People are filtering in, there is a delay before beginning, and Captain Last can choose where to go next.

			action
				"Hai Reveal [A11] Sound the Alarm: time" = 1
			# Randomly choose whether the proceedings or reading room are offered first to avoid consistently biasing players to either one.
			branch haidepart
				random < 50
			action
				set "Hai Reveal [A11] Sound the Alarm: proceedings first"
			label haidepart
			`	The functionary smiles sweetly as you approach and steps in close to lead you to a recessed doorway. Here, she is clearly audible above the growing background murmur. "So, where to, Captain?"`
			label wherego
			choice
				`	"What can I expect to find in this reading room?"`
					goto whazzit

> 				`	"I'll stay right here, I won't miss the proceedings."`

Reviewer 4: ```suggestion
				`	"I'll stay right here. I won't miss the proceedings."`
```
or
```suggestion
				`	"I'll stay right here and I won't miss the proceedings."`
```

					goto hurry
					to display
						has "Hai Reveal [A11] Sound the Alarm: proceedings first"
				`	"The reading room sounds very interesting. Lets go there."`
					goto reading

> 				`	"I'll stay right here, I won't miss the proceedings."`

Reviewer 4: ```suggestion
				`	"I'll stay right here. I won't miss the proceedings."`
```
```suggestion
				`	"I'll stay right here and I won't miss the proceedings."`
```

					goto hurry
					to display
						not "Hai Reveal [A11] Sound the Alarm: proceedings first"
				`	"Parliamentary reports and droning proceedings aren't my style. I'll catch up on sleep while I wait it out."`
			label "before you nap"
			`	"Oh, wow. Um... okay... Before you nap, the Elders have a predetermined question. How would you characterize Teeneep's meeting with the Unfettered Choot'k: constructive, aggressive, or something else?"`
			choice
				`	"It was a constructive discussion."`
				`	"I witnessed the test of two wills."`
				`	"The meeting was poignant."`
				`	"Very aggressive and confronting."`
				`	"Ugh. Fine. Take me to the proceedings."`
					goto nohurry
			`	The functionary takes you to another room, sets out a cot, and promises to wake you at the right time.`
				goto nap

			label whazzit
			action
				set "Hai Reveal [A11] Sound the Alarm: asked about files"

> 			`	"Oh, all manner of things. We try to keep it clean, but the Elders often stuff documents between cushions and leave hidden notes to each other. Once I found a datapad with a dossier on all interactions of our species - including the Unfettered - with other races. The Human, Quarg, Korath, and Wwww-" She twitches nervously, as if she caught herself before saying something she shouldn't. "-wwweeeellll, the room was cleaned an hour ago. Given the situation, I wouldn't be surprised if some Elder snuck something in with details about Teeneep's past, the Unfettered, and our interactions with human pirates." She smiles slyly.`

Reviewer 4: ```suggestion
			`	"Oh, all manner of things. We try to keep it clean, but the Elders often stuff documents between cushions and leave hidden notes to each other. Once I found a datapad with a dossier on all interactions of our species - including the Unfettered - with other races. The Humans, Quarg, Korath, and Wwww-" She twitches nervously, as if she caught herself before saying something she shouldn't. "-wwweeeellll, the room was cleaned an hour ago. Given the situation, I wouldn't be surprised if some Elder snuck something in with details about Teeneep's past, the Unfettered, and our interactions with human pirates." She smiles slyly.`
```
I assume Human is capitalized here intentionally?

			`	"I will, of course, relay their questions and your answers. Afterwards, our internal news network will give you the highlights of the proceedings. You can travel between the proceedings and the reading room at your leisure."`
			`	Her ears perk up as she smiles again, awaiting your response.`
				goto "interrupt rambling"

			# THE READING ROOM

			label reading

			# Entry points. We get here every time Captain Last leaves the proceedings, or upon first entering, or after each report is read.

			branch "no more time to read"
				"Hai Reveal [A11] Sound the Alarm: time" > 7
			branch "aimet is about to speak"
				"Hai Reveal [A11] Sound the Alarm: time" == 5
			branch "elders have a question"
				"Hai Reveal [A11] Sound the Alarm: time" == 3
			branch "decided to read nothing after seeing titles"
				"Hai Reveal [A11] Sound the Alarm: time" == 2
			branch "initial visit to reading room"
				"Hai Reveal [A11] Sound the Alarm: time" == 1

> 			branch library

Reviewer 4: No condition?

			label "initial visit to reading room"
			action
				"Hai Reveal [A11] Sound the Alarm: time" = 2
			`	The functionary delivers you to a comfortably appointed room with a handful of chairs and an adjustable height table. After assuring you that she'll come back when needed, she steps outside to join the guard posted by the door.`
			choice
				`	(Have a poke around.)`
				`	(Sit down and chill.)`
					goto beeps
			`	After fiddling with the table and sticking your hand down behind a couple of seat cushions, a seat on the other side of the room starts to give off a sharp, regular, beeping sound.`
				goto foundpad
			label beeps

> 			`	After a minute looking around and getting comfortable, you hear a sharp, regular, beep. It emanates from seat on the other side of the room.`

Reviewer 4: ```suggestion
			`	After a minute of looking around and getting comfortable, you hear a sharp, regular, beep. It emanates from seat on the other side of the room.`
```

			`	A voice keeps whispering something to you, but you can't quite understand. It seems to be coming from behind. A loud pain... a sharp pain... a furry pain...`
			choice
				`	"Who are you?"`
				`	"Get out of my head!"`
				`	(Turn around.)`
			`	You feel a claw poking repeatedly into your shoulder, startling you awake to the sight of an overgrown squirrel wearing a formal dress. She smiles down at you, making chittering noises that sound almost like words. After a few seconds of groggy confusion, you realize it is a Hai speaking. There was a functionary assigned to you during the proceedings, but that was so long ago... centuries... eons... the darkness begins to return...`
			`	"Captain Last! Wake up!" Her cheerful demeanor is a grating, but effective alarm clock.`
			label goto7

> 			`	"Captain <last>, the proceedings have completed, and your time here is at an end. The Elder Council voted not to punish Teeneep, probably because she is about to trigger an election. Some Elders have referred to her as Eyrisen of the Mythos," she giggles, "and if I was superstitious, I'd agree.`

Reviewer 4: ```suggestion
			`	"Captain <last>, the proceedings have completed, and your time here is at an end. The Elder Council voted not to punish Teeneep, probably because she is about to trigger an election. Some Elders have referred to her as Eyrisen of the Mythos," she giggles, "and if I was superstitious, I'd agree."`
```

						"Hai Reveal [A11] Sound the Alarm: time" > 2

			label solutions
			action
				set "Hai Reveal [A11] Sound the Alarm: solutions book"
				"Hai Reveal [A11] Sound the Alarm: time" ++
				log `A Hai intelligence report identifies three problems: "northern troubles," degrading secrecy of the Hai, and piracy. They expect piracy to vanish from the Republic "befriending" anarchist colonies and suggest Hai slowly reveal themselves through leaked information. It is unclear if that ever happened. For the "northern troubles," they have a "skilled mercenary" in mind for the job, and a diplomat.`
			`	This one-page report is recent, and includes references to other reports you can access.`
			# The "northern troubles" refers to the Unfettered raids on the Wanderers (who are to the north) not the troubles at Wah Ki (which is to the east). The Elders know, at some level, of the Unfettered-Wanderer hostilities. They're hoping to solve it through diplomacy (such as Sayari, INT-8C-131X04) or a skilled mercenary (such as Captain Last, INT-7Z-G81X51). In this mission, they gently nudge Captain Last to the north. They're using the captain's curious nature to lure the Captain into the conflict.

> 			`	"While we can all agree that the northern troubles could escalate to dangerous levels we've identified ways of improving relations through diplomacy [INT-8C-131X04] or a skilled mercenary [INT-7Z-G81X51]. It would appear that there are more immediate problems. Escalation of northern troubles is probably several decades away [UHI-001AX94], but total failure of the joint Hai-human disinformation campaign is expected in significantly less than that [HUM-9A38HF]. We must focus on the growing improbability of the secrecy of the Hai [HUM-931XHF9], and increasing pirate problems [HUM-9XC401Q]. Solving all three problems will open a path for Hai unification.`

Reviewer 4: ```suggestion
			`	"While we can all agree that the northern troubles could escalate to dangerous levels we've identified ways of improving relations through diplomacy [INT-8C-131X04] or a skilled mercenary [INT-7Z-G81X51]. It would appear that there are more immediate problems. Escalation of northern troubles is probably several decades away [UHI-001AX94], but total failure of the joint Hai-human disinformation campaign is expected in significantly less than that [HUM-9A38HF]. We must focus on the growing improbability of the secrecy of the Hai [HUM-931XHF9], and increasing pirate problems [HUM-9XC401Q]. Solving all three problems will open a path for Hai unification."`
```

> 			`	"Fortunately, there is increasing realization among the humans that the Pug 'War' was a benevolent act. We believe that, in the coming years, they'll be inclined view a new race in a favorable light. This merits a joint plan with human authorities to leak information that presents Hai as strong and benevolent [HUM-931XHF9].`

Reviewer 4: ```suggestion
			`	"Fortunately, there is increasing realization among the humans that the Pug 'War' was a benevolent act. We believe that, in the coming years, they'll be inclined view a new race in a favorable light. This merits a joint plan with human authorities to leak information that presents Hai as strong and benevolent [HUM-931XHF9]."`
```

> 			`	"The Free Worlds [HUM-9U4C12A] are assisting the Republic in a long-term solution to the pirate problem: befriend the planet's populations, and convince them to expel the pirates. We expect piracy in human space to reduce to minimal levels in ten years, except in eastern Syndicate space [HUM-8XK194C], where anarchist groups may have ties to the Korath [KTH-1C-910JX4C].`

Reviewer 4: ```suggestion
			`	"The Free Worlds [HUM-9U4C12A] are assisting the Republic in a long-term solution to the pirate problem: befriend the planet's populations, and convince them to expel the pirates. We expect piracy in human space to reduce to minimal levels in ten years, except in eastern Syndicate space [HUM-8XK194C], where anarchist groups may have ties to the Korath [KTH-1C-910JX4C]."`
```

			`	They can't track small pirate groups, but they clearly have an intelligence network. Is it good for anything?`
				goto reading
				to display
					not "Hai Reveal [A11] Sound the Alarm: human book"
			`	It seems that, although the Hai have no trouble with large, established hierarchies, they have trouble tracking small, independent, anarchist groups. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The Hai intelligence service depended on relatively stable conditions for their methods, which have faltered under conditions of asymmetric informational and organizational warfare.`
				goto reading
				to display
					has "Hai Reveal [A11] Sound the Alarm: human book"

			label unfettered
			action

> 				log `A Hai intelligence report puts the relative cost to Hai of seeking "full Hai unification" as being around 2.5 billion credits a day more expensive than handling the ongoing Unfettered raids`

Reviewer 4: ```suggestion
				log `A Hai intelligence report puts the relative cost to Hai of seeking "full Hai unification" as being around 2.5 billion credits a day more expensive than handling the ongoing Unfettered raids.`
```

			label next
			`	Sayari gives you a meaningful look and whispers, "A lot of them paid attention to that..."`
			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

> 			action
> 				"Hai Reveal [A11] Sound the Alarm: time" = 4
> 			choice
> 				`	(Remain silent and listen.)`
> 					goto "remain silent and listen"
> 				`	"That response was sickening. I'm going to the reading room."`
> 				`	"If you'll pardon me, I have some reports to read."`
> 			`	Before you can speak, Sayari puts her hand on your shoulder and shakes her head. "This is the moment, whatever happens next will decide what happens here," she whispers. "Be patient."`
> 			label "remain silent and listen"
> 			`	There is a growing rumble of assent as the crescendo of her impassioned statement crests to its conclusion. She levels a clawed finger directly at Teeneep and the room is filled with the angry cries of both those emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`

Reviewer 3: I think it makes more sense for choice to come after she points at Teeneep and is clearly finished.
```suggestion
			`	There is a growing rumble of assent as the crescendo of her impassioned statement crests to its conclusion. She levels a clawed finger directly at Teeneep and the room is filled with the angry cries of both those emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
			action
				"Hai Reveal [A11] Sound the Alarm: time" = 4
			choice
				`	(Remain silent and listen.)`
					goto "remain silent and listen"
				`	"That response was sickening. I'm going to the reading room."`
				`	"If you'll pardon me, I have some reports to read."`
			`	Before you can speak, Sayari puts her hand on your shoulder and shakes her head. "This is the moment, whatever happens next will decide what happens here," she whispers. "Be patient."`
			label "remain silent and listen"
```

			action
				"Hai Reveal [A11] Sound the Alarm: time" = 6
			choice
				`	"What are they saying?"`
				`	"What just happened?"`
				`	"If you'll pardon me, I have some reports to read."`
					goto reading
			`	She glances at you briefly. "They are saying: Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe, and despite the growing volume in the room she leans in close to you and continues in scarcely more than a whisper, "The Mythos has been invoked."`
			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator, who struggles to compose himself suitably. "I think the Council has heard enough. The petitioner is entitled to a closing statement, unless there is objection?"`
			`	None of those assembled who might object seem to have recovered their composure enough to stand up quickly, and the moderator, perhaps sensing a politic moment to intervene, quickly announces the closing of the interrogation and calls the Council to hear Teeneep again.`

> 			`	Teeneep quickly plasters back on her mask of calm as best she is able to and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`

Author: I have reworded it so it'll make sense.

> 			`	Teeneep quickly plasters back on her mask of calm as best she is able to and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`

Author: Actually, my mind is quite blurry right now, but it makes sense to me at the moment. What do you think?

> 			`	Teeneep quickly plasters back on her mask of calm as best she is able to and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`

Reviewer: It makes sense to me.

			`	He coughs again, and it echoes in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`
			`	The silence stretches with all eyes on Gemnali. She clenches her fists, shaking with barely suppressed emotion, before collapsing back into her seat as if physically struck.`
			`	Whatever spell had held the room is immediately broken and a susurrus of murmuring rises to fill its space. Within the noise a certain phrase starts to be repeated multiple times, drawing an expression of shock from Sayari.`

> 			action
> 				"Hai Reveal [A11] Sound the Alarm: time" = 6
> 			choice
> 				`	"What are they saying?"`
> 				`	"What just happened?"`
> 				`	"If you'll pardon me, I have some reports to read."`
> 					goto reading

Reviewer 3: ```suggestion
			action
				"Hai Reveal [A11] Sound the Alarm: time" = 6
			choice
				`	"What are they saying?"`
				`	"What just happened?"`
				`	"Excuse me, I need to check out something in the reading room."`
					goto reading
```

			`	There is a growing rumble of assent as the crescendo of her impassioned statement crests to its conclusion. She levels a clawed finger directly at Teeneep, inciting a cacophony. Elders emboldened by her diatribe and those distressed by its implication stir a storm of sound. The moderator's repeated attempts restore order fail as over a dozen Elders jump to their feet. At your side, Sayari's expression is ashen, but Teeneep looks around with fierce intensity. She seems to be seeking out a particular face somewhere in the room.`
			action
				"Hai Reveal [A11] Sound the Alarm: time" = 4
			choice
				`	(Remain silent and listen.)`
					goto "remain silent and listen"
				`	"That response was sickening. I'm going to the reading room."`
				`	"If you'll pardon me, I have some reports to read."`
			`	Before you can speak, Sayari puts her hand on your shoulder and shakes her head. "This is the moment, whatever happens next will decide what happens here," she whispers. "Be patient."`
			label "remain silent and listen"

> 			`	The sound of the room shifts and your attention is drawn to a particularly elderly Hai. He struggles to his feet in a series of pained stages, and effect on the assembly is profound. A ripple of silence flows outward from his position, and Elders all around promptly drop back into their seats as it reaches them. A scant handful who were too slow to react have their final words trail off awkwardly. Only the last speaker remains on her feet. The moderator weakly announces in Hai what you know to be, "The council hears Aimet."`

Reviewer 4: ```suggestion
			`	The sound of the room shifts and your attention is drawn to a particularly elderly Hai. He struggles to his feet in a series of pained stages, and effect on the assembly is profound. A ripple of silence flows outward from his position, and Elders all around promptly drop back into their seats as it reaches them. A scant handful who were too slow to react have their final words trail off awkwardly. Only the last speaker remains on her feet. The moderator weakly announces in Hai words you know to be, "The council hears Aimet."`
```

				goto continue4
			label return4
			`	On the short walk to the proceedings, the functionary explains, "A firebrand named Gemnali has made serious accusations against Teeneep."`

> 			`	You can hear the tail end of a diatribe, as only one last speaker remains on her feet. The room has fallen to an awed silence as the the moderator weakly announces in Hai, "The council hears Aimet."`

Reviewer 4: You don't get a description of Aimet here.

			label continue4
			action
				"Hai Reveal [A11] Sound the Alarm: time" = 5

> 			`	"Security through secrecy..." He coughs. "You throw that in there as if it were always so Gemnali. Unlike you, I was there when it was added to the mandate of this chamber."`

Reviewer 4: ```suggestion
			`	"Security through secrecy..." He coughs. "You throw that in there as if it were always so, Gemnali. Unlike you, I was there when it was added to the mandate of this chamber."`
```

> 			`	He peers around the room, and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter... You would have us pursue secrecy at the cost of security."`

Reviewer 4: ```suggestion
			`	He peers around the room, and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter... you would have us pursue secrecy at the cost of security."`
```

			`	The silence stretches with all eyes on Gemnali. She clenches her fists, shaking with barely suppressed emotion, before collapsing back into her seat as if physically struck.`
			`	Whatever spell had held the room is immediately broken and a susurrus of murmuring rises to fill its space. Within the noise a certain phrase starts to be repeated multiple times, drawing an expression of shock from Sayari.`
			action
				"Hai Reveal [A11] Sound the Alarm: time" = 6
			choice
				`	"What are they saying?"`
				`	"What just happened?"`
				`	"Excuse me, I need to check out something in the reading room."`
					goto reading
			`	She glances at you briefly. "They are saying: Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe. Despite the growing volume in the room she leans in close to you and continues in scarcely more than a whisper, "The Mythos has been invoked."`

> 			`	Aimet painfully resumes his seat and another Elder stands. This Elder had been maintaining a fairly neutral demeanor, but appears to gain a sense of himself. He is quickly heralded by the moderator, who struggles to compose suitably. "I think the Council has heard enough. The petitioner is entitled to a closing statement, unless there is objection?"`

Reviewer 4: Sense of what? Pride?

			`	None of those assembled who might object seem to have recovered their composure enough to stand up quickly. The moderator, perhaps sensing a politic moment to intervene, quickly announces the closing of the interrogation. He calls the Council to hear Teeneep again.`
			`	Teeneep quickly plasters on a mask of calm as best she is able and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`

> 			`	One Elder cries out, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`

Reviewer 4: I know it's not a line that is already _directly_ changed by this PR, but seeing as how the subject of this PR is about this mission...
```suggestion
			`	One Elder cries out, "You wouldn't dare!" to which Teeneep immediately responds, "It is already scheduled to happen unless I actively stop it!"`
```

> 			`	Before any further exchange can occur, the moderator bellows for security. Guards escort you, Teeneep, Sayari, and the functionary, from the session. The moderator shouts down the offending Elder for impropriety and the two engage in a lively shouting match. Its echoes follow you until the chamber doors are shut.`

Reviewer 4: ```suggestion
			`	Before any further exchange can occur, the moderator bellows for security. Guards escort you, Teeneep, Sayari, and the functionary from the session. The moderator shouts down the offending Elder for impropriety and the two engage in a lively shouting match. Its echoes follow you until the chamber doors are shut.`
```

			# Proceedings end.
			action
				"Hai Reveal [A11] Sound the Alarm: time" = 7
			`	Teeneep moves to pull out a datapad, but both the security officers and Sayari step in to stay her hand. "The chamber will have to vote on the proposition to act," says Sayari. She looks at Teeneep pointedly. "You know that external contact is forbidden during the deliberation grace window."`
			choice
				`	"The deliberation grace window?"`
				`	"Just wait? I'll wait in the reading room."`
					goto reading
				`	"Why are we waiting?"`
					goto wait
			`	Teeneep sighs and points an exasperated expression your way. "Yes, after certain types of chamber hearings there is an enforced period. The petitioner must be secluded in the chamber's waiting area. I forgot for a moment."`
			label wait
			`	Sayari smiles at you. "Don't worry though, it's only for about twenty-three of your minutes." She eyes the security officer warily. "After a session such as that though, the traditional wisdom is to wait in relative silence."`

> 			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be so didn't plan for it."`

Reviewer 4: ```suggestion
			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be this way and so didn't plan for it."`
```

			
			`	After only about seventeen minutes the moderator slips out and exchanges some words with Teeneep and Sayari before re-entering the chamber. "We've been released," says Sayari. "They've voted to take no immediate action, either for or against Teeneep."`
			`	Teeneep just shakes her head, pulls out a datapad, and quickly taps a few times. "Well, that was going to happen in an hour. No point waiting, though. Let's get dinner."`
			choice
				`	"Actually, I'll go watch the summary of the proceedings."`
				`	"Dinner sounds great to me."`
					goto dinner
				`	"No, thank you. I need to check in with the diplomats.`
					goto diplomats
			`	Teeneep gives you directions to a common area with a large holoprojector. You arrive to a room filled with Hai government employees. From your limited experience, they look to have medium levels of clearance. Enough to need a summary, without getting to witness sensitive specifics. You arrive just as the 'news' is about to begin.`

			# SCENE CHANGE

			# Deliberation grace period is over and everybody leaves the chamber.

			label news

> 			`	The holoprojector turns on, and an elderly Hai presents. "In today's closed emergency meeting conducted in the human language, the full Elder Council put the known disruptor, Teeneep, on trial. She was charged with risking the safety of all Hai by her recent actions. While she made an eloquent defense, her previous confession was considered enough evidence to prove her guilt. There was audibly wide-spread support among the Elders for a stern and righteous action in defense of the secrecy mandate... until that very premise was challenged.`

Reviewer 4: Presents what? That first sentence might need a bit more context.

			`	"In a rare utterance, Aimet Taepip silenced the Council to speak in favor of Teeneep, and against the secrecy mandate. He presented a bold reminder of recent history and a stark choice. Secrecy and safety are now conflict; we must choose one or the other. The scene inspired certain sections to invoke the mythos, 'Veeheylur and Misanaya fight over Eyrisen'. This event appears to have shaken many members of the Council who had previously declared positions. In the vote immediately following, enough Elders abstained that the matter could not be progressed.`

> 			`	"Today's decision... was indecision. A watch is being placed on a petition identified in association with this that may reach its threshold in weeks. An election trigger will be the outcome and all departments are on alert to take appropriate action."`

Reviewer 4: I'm not sure ellipses are needed here.

> 			`	"Briefing will repeat in Hai in 1 minute."`

Reviewer 4: ```suggestion
			`	"Briefing will repeat in Hai in one minute."`
```

> 			`	A Hai next to you tells you, "It's all a bit disconcerting, isn't it? Pirates and Unfettered on our borders and Elders doing nothing. A chaos magnet causing mischief while a human mercenary helps her. I wish I was there to see the proceedings, but I know they weren't recorded."`

Reviewer 4: "tells" doesn't work if the first sentence is a question.
```suggestion
			`	A Hai next to you says to you, "It's all a bit disconcerting, isn't it? Pirates and Unfettered on our borders and Elders doing nothing. A chaos magnet causing mischief while a human mercenary helps her. I wish I was there to see the proceedings, but I know they weren't recorded."`
```

			choice
				`	"Well, you just have to keep living your life, right?"`
				`	"I'm sure the mercenary will fix all of this."`
				`	"Political discussions are always droll, regardless of their consequences."`

> 			`	As he shrugs and walks away, you realize your acute hunger. Foraging for food in a nearby cafeteria, you soon find Alondo and Remington. They have a surprisingly civilized conversation the entire time. Albeit mostly about work.`

Reviewer 4: ```suggestion
			`	As he shrugs and walks away, you realize your acute hunger. Foraging for food in a nearby cafeteria, you soon find Alondo and Remington. They have a surprisingly civilized conversation the entire time, albeit mostly about work.`
```

			choice
				`	"Yes, I certainly want the summary."`
				`	"No, it's time for dinner."`
					goto spydinner
			`	"As you wish," she says. After a minute or two listening to her chat about the importance of Teeneep's work, you arrive at a common area with a holoprojector. Before she departs, her face turns dead serious and she whispers a warning. "After this all blows over, you'll be needed up north. Keep your eyes in that direction, Captain; many lives will depend on it."`
			`	You open your mouth to respond, but she quickly vanishes into the crowd. Surrounding you are officers, functionaries, and diplomats, all awaiting the news briefing that is about to begin.`
				goto news
			label spydinner
			`	"As you wish," she says. After a minute or two listening to her chat about the importance of Teeneep's work, you arrive at a cafeteria. She helps you find the best dishes, and leads you to the seating area. Before she departs, her face turns dead serious and she whispers a warning. "After this all blows over, you'll be needed up north. Keep your eyes in that direction, Captain; many lives will depend on it." You open your mouth to respond, but she quickly vanishes into the crowd.`

> 			`	You hear Alondo and Remington behind you; one of them asks you to sit and eat with them. The dinner is quite pleasant and they have a surprisingly civilized conversation the entire time. Albeit mostly about work.`

Reviewer 4: ```suggestion
			`	You hear Alondo and Remington behind you; one of them asks you to sit and eat with them. The dinner is quite pleasant and they have a surprisingly civilized conversation the entire time, albeit mostly about work.`
```

				goto panic

			label diplomats

> 			`	There are some perfectly adequate meal places in the cafeteria area. There you manage to join Alondo and Remington, who have a surprisingly civilized conversation the entire time. Albeit mostly about work.`

Reviewer 4: ```suggestion
			`	There are some perfectly adequate meal places in the cafeteria area. There you manage to join Alondo and Remington, who have a surprisingly civilized conversation the entire time, albeit mostly about work.`
```

			`	After only about seventeen minutes the moderator slips out and exchanges some words with Teeneep and Sayari before re-entering the chamber. "We've been released," says Sayari. "They've voted to take no immediate action, either for or against Teeneep."`
			`	Teeneep just shakes her head, pulls out a datapad, and quickly taps a few times. "Well, that was going to happen in an hour. No point waiting, though. Let's get dinner."`
			choice
				`	"Actually, I'll go watch the summary of the proceedings."`
				`	"Dinner sounds great to me."`
					goto dinner
				`	"No, thank you. I need to check in with the diplomats.`
					goto diplomats
			`	Teeneep gives you directions to a common area with a large holoprojector. You arrive just as the news is about to begin.`

> 
> 			# SCENE CHANGE
> 
> 			# Deliberation grace period is over and everybody leaves the chamber.
> 
> 			label news
> 			`	The holoprojector turns on, and an elderly Hai presents the news. "In today's closed emergency meeting, the full Elder Council put the known disruptor, Teeneep, on trial for risking the safety of all Hai by her recent actions. While she made an eloquent defense, her previous confession was enough evidence to prove her guilt. There was audibly wide-spread support among the Elders for a stern and righteous action in defense of the secrecy that is so key to our security... until that very premise was challenged.`
> 			`	"All the Council was silenced in awe as Aimet Taepip spoke in favor of Teeneep, against the secrecy doctrine, presenting a stark choice. Secrecy and safety now conflict; we must choose one or the other: Teeneep's path to safety, or Gemnali's path to secrecy. At this, some brought to attention what traditionalists have said for years: the remarkable similarity between Teeneep's life and that of Eyrisen from the Mythos. The Mythos and Aimet together convinced many Elders to back down, but not enough.`
> 			`	"Today's decision... was indecision. We await another path through the chaos that slowly engulfs us. There are rumors of a petition that may reach its threshold in weeks. Perhaps an election will bring us rapidly to our salvation or doom before time runs out."`
> 			`	A Hai next to you tells you, "It's all a bit disconcerting, isn't it? Pirates and Unfettered on our borders and Elders doing nothing. A chaos magnet causing mischief while a human mercenary helps her. I wish I was there to see the proceedings, but I know they weren't recorded."`
> 			choice
> 				`	"Well, you just have to keep living your life, right?"`
> 				`	"I'm sure the mercenary will fix all of this."`
> 				`	"Political discussions are always droll, regardless of their consequences."`
> 			`	As he shrugs and walks away, you realize your acute hunger. Foraging for food in a nearby cafeteria, you soon find Alondo and Remington. They have a surprisingly civilized conversation the entire time. Albeit mostly about work.`
> 				goto panic

Reviewer 3: The implication of this whole sequence btw, is that this is *all* meant to be restricted, non-public information - but by virtue of the mythos having been invoked, it is simply uncontainable. People moved by superstitious beliefs of fate and destiny will share those when dealing with such matters because it is fundamental to them, and so it is that power that drives the spread of information from there.
Also, alignment for briefing format.
```suggestion
			# SCENE CHANGE

			# Deliberation grace period is over and everybody leaves the chamber.

			label news
			`	The holoprojector turns on, and an elderly Hai presents. "In today's closed emergency meeting conducted in the human language, the full Elder Council put the known disruptor, Teeneep, on trial for risking the safety of all Hai by her recent actions. While she made an eloquent defense, her previous confession was considered enough evidence to prove her guilt. There was audibly wide-spread support among the Elders for a stern and righteous action in defense of the secrecy mandate... until that very premise was challenged.`
			`	"In a rare utterance Aimet Taepip silenced the Council to speak in favor of Teeneep, and against the secrecy mandate, presenting a bold reminder of recent history and a stark choice. Secrecy and safety are now conflict; we must choose one or the other. The scene inspired certain sections to invoke the mythos, 'Veeheylur and Misanaya fight over Eyrisen'. This event appears to have shaken many members of the Council who had previously declared positions. In the vote immediately following, enough Elders abstained that the matter could not be progressed.`
			`	"Today's decision... was indecision. A watch is being placed on a petition identified in association with this that may reach its threshold in weeks. An election trigger will be the outcome and all departments are on alert to take appropriate action."`
			`	"Briefing will repeat in Hai in 1 minute."`
			`	A Hai next to you tells you, "It's all a bit disconcerting, isn't it? Pirates and Unfettered on our borders and Elders doing nothing. A chaos magnet causing mischief while a human mercenary helps her. I wish I was there to see the proceedings, but I know they weren't recorded."`
			choice
				`	"Well, you just have to keep living your life, right?"`
				`	"I'm sure the mercenary will fix all of this."`
				`	"Political discussions are always droll, regardless of their consequences."`
			`	As he shrugs and walks away, you realize your acute hunger. Foraging for food in a nearby cafeteria, you soon find Alondo and Remington. They have a surprisingly civilized conversation the entire time. Albeit mostly about work.`
				goto panic
```

			label return7
			`	After a minute of walking, you arrive at an empty room. "It seems Teeneep and Sayari have departed to have private discussions, but I can tell you where to meet with the human diplomats. If you prefer, I can take you to a summary of the proceedings instead."`
			choice
				`	"Yes, I certainly want the summary."`
				`	"No, it's time for dinner."`
					goto spydinner
			`	"As you wish," she says. After a minute or two listening to her chat about the importance of Teeneep's work, you arrive at a common area with a holoprojector. Before she departs, her face turns dead serious and she whispers, "After this all blows over, you'll be needed up north. Keep your eyes in that direction, Captain; many lives will depend on it."`

> 			`	You open your mouth to respond, but she quickly vanishes into the crowd. Surrounding you are Elders, functionaries, and diplomats, all awaiting the news report that is about to begin.`

Reviewer 3: ```suggestion
			`	You open your mouth to respond, but she quickly vanishes into the crowd. Surrounding you are officers, functionaries, and diplomats, all awaiting the news briefing that is about to begin.`
```

				goto news
			label spydinner

> 			`	"As you wish," she says. After a minute or two listening to her chat about the importance of Teeneep's work, you arrive at a cafeteria. She helps you find the best dishes, and leads you to the seating area. Before she departs, her face turns dead serious and she whispers, "After this all blows over, you'll be needed up north. Keep your eyes in that direction, Captain; many lives will depend on it." You open your mouth to respond, but she quickly vanishes into the crowd.`

Reviewer 3: And now, with the changes above, if you take this route and only this route, does the hand that holds the knife reveal itself. It does so because, by having skipped the events to nap, you have now become the knife.
